CMakeLists.txt

```diff
@@ -744,17 +744,39 @@ add_subdirectory(packaging)
 set(CPACK_PACKAGE_DIRECTORY "dist")
 set(CPACK_PACKAGE_FILE_NAME "isle-${PROJECT_VERSION}-${ISLE_PACKAGE_NAME}")
 if(NINTENDO_3DS)
+  find_program(BANNERTOOL bannertool)
+  find_program(MAKEROM makerom)
+
   ctr_generate_smdh(isle.smdh
     NAME "LEGO Island"
     TITLE "LEGO Island"
     DESCRIPTION "LEGO Island for the Nintendo 3DS"
     AUTHOR "isledecomp/isle-portable"
     VERSION "${PROJECT_VERSION}"
-    ICON "ISLE/res/3ds/isle.png"
+    ICON "ISLE/res/3ds/icon.png"
   )
 
   ctr_create_3dsx(isle SMDH isle.smdh)
+  if(BANNERTOOL AND MAKEROM)
+    add_custom_command(
+      OUTPUT "isle.bnr"
+      COMMAND "${BANNERTOOL}" makebanner -i "../ISLE/res/3ds/banner.png" -a "../ISLE/res/3ds/banner.wav" -o "isle.bnr"
+      DEPENDS "../ISLE/res/3ds/banner.png" "../ISLE/res/3ds/banner.wav"
+      VERBATIM
+    )
+
+    add_custom_command(
+      OUTPUT "isle.cia"
+      COMMAND "${MAKEROM}" -f cia -exefslogo -o "isle.cia" -rsf "../ISLE/res/3ds/template.rsf" -major "${CMAKE_PROJECT_VERSION_MAJOR}" -minor "${CMAKE_PROJECT_VERSION_MINOR}" -micro 0 -icon "isle.smdh" -banner "isle.bnr" -elf "isle.elf"
+      DEPENDS "ISLE/res/3ds/template.rsf" "isle.smdh" "isle.bnr"
+      COMMENT "Building CIA executable target isle.cia"
+      VERBATIM
+    )
```

Comment:
Please split the `makerom` command over multiple lines. Right now, the line is too long.

Also, please use absolute paths, if possible.